src/main/java/thePackmaster/actions/bitingcoldpack/FrostbiteDamageAction.java

```diff
@@ -0,0 +1,68 @@
+package thePackmaster.actions.bitingcoldpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
+import com.megacrit.cardcrawl.actions.utility.WaitAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+import thePackmaster.powers.bitingcoldpack.ColdSeasonPower;
+import thePackmaster.powers.bitingcoldpack.FrostbitePower;
+import thePackmaster.relics.bitingcoldpack.Snowglobe;
+
+import static thePackmaster.util.Wiz.applyToEnemyTop;
+
+public class FrostbiteDamageAction extends AbstractGameAction {
+    private final DamageInfo info;
+    private int damage;
+    private final AbstractPower callingPower;
+    private boolean isItColdSeason = false;
+
+    public FrostbiteDamageAction(AbstractCreature target, DamageInfo info, AbstractPower callingPower) {
+        this.info = info;
+        this.damage = this.info.base;
+        this.callingPower = callingPower;
+        setValues(target, info);
+        this.actionType = AbstractGameAction.ActionType.DAMAGE;
+        this.duration = this.startDuration = Settings.ACTION_DUR_FAST;
+    }
+
+    @Override
+    public void update() {
+        if(shouldCancelAction()) {
+            this.isDone = true;
+        } else {
+            tickDuration();
+            if (this.isDone) {
+
+                // Snowglobe
+                if (AbstractDungeon.player.hasRelic(Snowglobe.ID)) {
```

Comment:
Just a small tip, you can store the getRelic return in an AbstractRelic variable and check if it's null. This saves you a list iteration through player.relics